opts/mount_test.go

```diff
@@ -28,6 +29,22 @@ func TestMountOptString(t *testing.T) {
 	assert.Check(t, is.Equal(expected, mount.String()))
 }
 
+func TestMountRelative(t *testing.T) {
+	var mount MountOpt
+
+	assert.NilError(t, mount.Set("type=bind,source=.,target=/target"))
```

Comment:
Can you add test-cases for `./` as well? Might be worth creating an array/table with test-cases (and using sub-tests) if that helps making it less DRY.

Reply:
Done